week2/Week_2 (DraftForClass).ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<p>It's not clear to me why you are creating a pandas df from a numpy array. It's not wrong, it just seems a little redundant. I think just having the numpy array is good. Makes the math cleaner too. </p><p> </p><p> </p><p>Why are you looking for the slope? What are you trying to plot?</p><p>Consider plotting distances that are computed from velocity and time. Might be clearer for vector illustration.</p><p>Don't forget a title. Something like 'Boat on windless lake moving 50 m/s at 60deg CCW (NE)'</p><p>Figure size is a little small, text a bit hard to read. Recommend something a bit bigger for laptop screens.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>